integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -439,18 +439,11 @@ fuse:
   user: 0
   group: 0
   fsGroup: 0
-  # fuse args specific to each standalone fuse
-  args:
-    - fuse
-    - --fuse-opts=allow_other
-  # Default fuse mount options shared by all fuse applications
-  # Overwrite by entrypoint.sh fuse --fuse-opts <fuse_mount_options>
```

Comment:
There is little reason to tell user how to use them for entrypoint.sh because users shall not interact with entrypoint.sh